.travis.yml

```diff
@@ -43,7 +43,7 @@ env:
   # Linting is broken. Disable until fixed.
   # See https://github.com/bazelbuild/rules_scala/pull/622
   # we want to test the last release
-  #- TEST_SCRIPT=test_lint.sh
+  - TEST_SCRIPT=test_lint.sh
```

Comment:
can we avoid running the lint on all platforms? Is that easy?

Reply:
Yep! I'm going to rework travis so we use the more configurable job pipelines instead of the strict square matrix.

Reply:
I think we should test linting via buildkite. See this readme for more [info] (https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/README.md#running-buildifier-on-ci)